### ISSUE TITLE
Missing Labels for table filters [Accessibility ⭐ ]

### ISSUE THREAD

**Issue by sumairq** [mozilla/perfcompare#413]
The filter buttons don't have accessible labels, as a result the filters are not read properly by screen readers.
